src/robot/writer/rowsplitter.py

```diff
@@ -75,7 +75,10 @@ def _split_row(self, row, indent):
             current, row = self._split(row)
             yield self._escape_last_cell_if_empty(current)
             if row:
-                row = self._continue_row(row, indent)
+                if self._comment_mark in current:
```

Comment:
I checked it and the answer is no, it wouldn't break it. But there were other problems in my code that I fixed in a new commit. Please see the updated pull request.